buildkite/scripts/fuzzy-zkapp-test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -eo pipefail
+
+if [[ $# -ne 2 ]]; then
+    echo "Usage: $0 <dune-profile> <path-to-source-tests>"
+    exit 1
+fi
+
+profile=$1
+path=$2
+
+source ~/.profile
+
+echo "--- Make build"
+export LIBP2P_NIXLESS=1 PATH=/usr/lib/go/bin:$PATH GO=/usr/lib/go/bin/go 
+time make build
```

Comment:
If this is intended to run in CI, why make build? many other jobs exist to build the daemon